modules/eks/actions-runner-controller/README.md

```diff
@@ -31,21 +31,30 @@ components:
         chart_version: "0.20.2"
         kubernetes_namespace: "actions-runner-system"
         create_namespace: true
+        kubeconfig_exec_auth_api_version: "client.authentication.k8s.io/v1beta1"
+        # Stay within the limits of c7g.xlarge, defined by karpenter
+        # vCPU 4, Memory 8 GiB
         resources:
           limits:
-            cpu: 50m
-            memory: 100Mi
+            cpu: 4
+            memory: 8Gi
           requests:
-            cpu: 10m
-            memory: 30Mi
-        ssm_github_token_path: "/github/acme/github_token"
-        ssm_github_webhook_secret_token_path: "/github/acme/github_webhook_secret_token"
+            cpu: 1
+            memory: 2Gi
```

Comment:
These limits seem ridiculously high for the runner-controller. All it does is launch and reap runners. The previous limits seem about right to me.

Reply:
these limits were highly raised for the GitHub Actions release workflows. The `app-runner` is building customer app images as part of that workflow. The `infrastructure-runner` can be a lot lower though

Reply:
These are limits for the runner _controller_, not the runner. All the runner controller does is launch and reap runners.